torchmdnet/datasets/hdf.py

```diff
@@ -57,6 +57,15 @@ def __init__(self, filename, dataset_preload_limit=1024, **kwargs):
                             self.fields.append(
                                 ("partial_charges", "partial_charges", torch.float32)
                             )
+                        # total charge and spin, will be load as 'q' and 's' respectively to keep the same naming convention
```

Comment:
This comment does not seem to be enforced later on.
What happens if I have an old hdf around with "q" and "qs" fields?

Reply:
You should open it and rename the datasets with the allowed labels, i.e. `total_charge` and `spin`. Otherwise this dataloader will be if/elif/else

Reply:
But this is contradicting the comment, which says that those properties will be loaded as "q" and "s"?

Reply:
yes, it's a residue. I'm removing it